webapp/tests/karma/ts/services/enketo.service.spec.ts

```diff
@@ -504,10 +525,21 @@ describe('Enketo service', () => {
   describe('save', () => {
 
     it('rejects on invalid form', done => {
```

Comment:
Can you please convert this test to not use `done` and instead return a promise?
There's a really poor experience when the test fails, where it actually reports a timeout (when done is not called) instead of the assertion error.